src/main/java/net/sf/jabref/gui/groups/GroupDialog.java

```diff
@@ -296,27 +280,13 @@ public void actionPerformed(ActionEvent e) {
                 mOkPressed = true;
             try {
                 if (m_explicitRadioButton.isSelected()) {
-                    if (m_editedGroup instanceof ExplicitGroup) {
-                        // keep assignments from possible previous ExplicitGroup
-                        mResultingGroup = m_editedGroup.deepCopy();
-                        mResultingGroup.setName(m_name.getText().trim());
-                        mResultingGroup.setHierarchicalContext(getContext());
-                    } else {
-                        mResultingGroup = new ExplicitGroup(m_name.getText().trim(), getContext());
-                        if (m_editedGroup != null) {
-                            addPreviousEntries();
-                        }
-                    }
+                    mResultingGroup = new ExplicitGroup(m_name.getText().trim(), getContext());
```

Comment:
Since you are touching that code anyway: How about refactoring the names here according to Java code conventions? `m_name` -> `mName`. What does the `m` stand for, btw?